templates/components/streamfield/blocks/card_block.html

```diff
@@ -0,0 +1,17 @@
+{% verbatim %}
+<div class="flex flex-col">
+    {% if value.heading %}
+      <h3 class="mb-8 text-xl md:text-2xl text-mackerel-300 font-semibold">{{ value.heading }}</h3>
+    {% endif %}
+    {% if value.description %}
+      <div class="mb-8 md:mb-12 md:pb-16 text-lg md:text-xl">
```

Comment:
Some of the classes in this PR (not sure which ones) don't yet exist in the generated Tailwind CSS, so we need to run `npm run build:prod`. I'll do that before merging.